addons/packages/vsphere-pv-csi/2.4.1/bundle/config/values.yaml

```diff
@@ -5,5 +5,3 @@ vspherePVCSI:
   namespace: "vmware-system-csi"
   supervisor_master_endpoint_hostname: "supervisor.default.svc"
   supervisor_master_port: 6443
-  cluster_uid: null
```

Comment:
I know removing these 2 params will make schema non-null enforce work. But we are treating the values.yaml inside the package as a reference for all the available configs, wondering if this is best practice to remove these 2 values.

Reply:
Using values.yaml as reference is not the best idea, we should be using docs or sample-values. Ideally speaking these two fields should not something that the package requires.

For a better example consider grafana package which has a password field which we would want to enforce as a required param. Making it an empty string and putting it values.yaml would defeat the intention of the schema.